pkg/operator/ceph/csi/spec.go

```diff
@@ -378,6 +379,10 @@ func startDrivers(namespace string, clientset kubernetes.Interface, ver *version
 		// apply resource request and limit to rbdplugin containers
 		applyResourcesToContainers(clientset, rbdPluginResource, &rbdPlugin.Spec.Template.Spec)
 		k8sutil.SetOwnerRef(&rbdPlugin.ObjectMeta, ownerRef)
+		err = applyCephClusterNetworkConfig(&rbdPlugin.Spec.Template.ObjectMeta, rookclientset)
```

Comment:
The annotations are applied to the csi daemonset pods but the `k8s.v1.cni.cncf.io/networks-status` annotation is missing so couldn't verify if the pods are using multus

Reply:
Is the annotation getting overwritten?

Reply:
No.
Just that there is no `k8s.v1.cni.cncf.io/networks-status` annotation in the pod and I am not able to get in the containers to check if the multus interfaces are added

Reply:
The annotations are applied normally

Reply:
Well how can the annotation be applied normally if not visible later on? Something is off.

Reply:
The annotation is applied
```
annotations:
    k8s.v1.cni.cncf.io/networks: public
    openshift.io/scc: rook-ceph-csi
```

There should be a `k8s.v1.cni.cncf.io/networks-status`
something like the following:
```
Annotations:  k8s.v1.cni.cncf.io/networks: public
              k8s.v1.cni.cncf.io/networks-status:
                [{
                    "name": "openshift-sdn",
                    "interface": "eth0",
                    "ips": [
                        "10.128.2.31"
                    ],
                    "dns": {},
                    "default-route": [
                        "10.128.2.1"
                    ]
                },{
                    "name": "public",
                    "interface": "net1",
                    "ips": [
                        "192.168.1.209"
                    ],
                    "mac": "4e:5d:a8:b7:36:2b",
                    "dns": {}
                }]
              openshift.io/scc: rook-ceph-csi```